3604/1/top_gems.rb

```diff
@@ -0,0 +1,81 @@
+require 'terminal-table'
+require './data_initializer.rb'
+require './get_information_about_gem.rb'
+require './create_rating.rb'
+
+# class for main task: start project, get array of names and get arguments
+# return table with all information
+class TopGems
+  def initialize
+    @array_gems = []
+    @temp_arr = ['', ' used by', ' watched by', ' stars', ' forks', ' contributors', ' issues']
```

Comment:
there is a special place in hell for people who write full of temp variables. 
If you can't give a name to a variable then you definitely don't understand why you need it and what you do

Reply:
Looks like a constant. Why is it here?